.evergreen-functions.yml

```diff
@@ -490,12 +490,53 @@ functions:
           - rh_pyxis
         binary: scripts/dev/run_python.sh scripts/preflight_images.py --image ${image_name} --submit "${preflight_submit}"
 
+  install_macnotary:
```

Comment:
do we still need all the other places where we access and install the notary? It seems we have these functions here but also other usages

Reply:
Sorry, I didn't really understand this. I am consuming this function from the places where ever we are building the tests image the examples are in the `.evergreen.yml` file.

Reply:
> do you plan to remove it? Is it tracked? If not we should do so.

Sure. I have not created the jira tickets, I will create one.

> aren't we using build_multi_cluster_kubeconfig_creator.sh locally as well to build the image? Is that still required?

Yes, it's used via make target `build-multi-cluster-binary` if someone wants to build the binary or tests image locally. I will create a ticket to use goreleaser for that as well and then we just just get rid of `build_multi_cluster_kubeconfig_creator.sh`.

There is one more place where we build kubectl plugin separately, to use it for `make prepare-local-e2e`, we should eventually move away from that as well. But that's not in scope of this PR/project.

Reply:
my problem of the current approach is the duplication on how we create the binary. In some point of time we will have a limbo of atomics way via the python script and local way via the bash script. If one changes and a workflow breaks we will need to manually debug to find out the cause. 

That happened too often and we should not do this - again.

IMO - we should do all together now - or in a set of tickets clearly planned to be done